beta/nncf/tensorflow/graph/converter.py

```diff
@@ -234,7 +238,7 @@ def _update_graph_with_raw_nodes(graph: Union[nx.DiGraph, NNCFGraph],
     for original_name, instances in raw_nodes.items():
         for i, attributes in instances.items():
             node_name = get_expanded_node_name(original_name, i, attributes['is_shared'])
-            graph.add_node(node_name, **attributes)
+            graph.add_node(node_name, original_name=original_name, **attributes)
```

Comment:
Why do you use the keyword argument?

Reply:
In case of shared layers (when one layer calls several time), node_name is 'nncf_wrapped_conv1^0', 'nncf_wrapped_conv1^1'.
To find corresponding nncf_node we should know original_name,

Reply:
I mean, why we can not write something like this

```suggestion
            graph.add_node(node_name, original_name, **attributes)
```

Reply:
`graph.add_node(node_name, original_name=original_name, **attributes)` is short analog 
```
attributes['original_name'] = original_name
graph.add_node(node_name, **attributes)
```

And `graph.add_node(node_name, original_name, **attributes)`  looks like TypeError, because 
```
def add_node(self, node_for_adding, **attr):
```

Reply:
Ok, thanks.